docs/agents/sematext-agent/releasenotes.md

```diff
@@ -39,6 +39,20 @@ NULL
 
 -->
 
+## Version 4.3.1
+
+Date: May 15, 2024
+
+### Improvements
+
+- Support for monitoring services running on the Containerd runtime in Kubernetes has been added.
```

Comment:
Is this about the JMX issue? If yes, the text is TOO generic, it sounds like we start supporting containerd today